plugins/woocommerce/tests/e2e-pw/tests/merchant/users-manage.spec.js

```diff
@@ -65,7 +65,11 @@ async function userDeletionTest( page, username ) {
 			page.getByRole( 'heading', { name: 'Delete Users' } )
 		).toBeVisible();
 
-		await expect( page.getByText( `${ username }` ) ).toBeVisible();
+		await expect(
+			page
+				.getByText( 'Delete Users You have' )
+				.getByText( `${ username }` )
+		).toBeVisible();
```

Comment:
Make it more unique. Without this change, on WPCOM, it is resolved to three elements.